README.rst

```diff
@@ -545,6 +545,8 @@ Choose from the list of available rules:
 
   Configuration options:
 
+  - ``ensure_fully_multiline`` (``bool``): ensure every argument of a multiline
```

Comment:
Done. I also appended the rule as described in the PSR-2 guide in the description